### PR TITLE
fix demo_ci bug in vis_demo.cc

### LINKED ISSUE #13841: manylinux1 fails on TRTConfig in demo_ci
Three manylinux1 version (cpu_avx_openblas, cpu_noavx_openblas, cuda8.0_cudnn5_avx_openblas) fails on TRTConfig in demo_ci
http://ci.paddlepaddle.org/viewLog.html?buildId=22527&buildTypeId=Manylinux1_CpuNoavxOpenblas&tab=buildLog
```
[01:56:50]	[Step 5/5] CMakeFiles/vis_demo.dir/vis_demo.cc.o: In function `paddle::demo::Main(bool, bool)':
[01:56:50]	[Step 5/5] vis_demo.cc:(.text+0x13f7): undefined reference to `std::unique_ptr<paddle::PaddlePredictor, std::default_delete<paddle::PaddlePredictor> > paddle::CreatePaddlePredictor<paddle::contrib::MixedRTConfig>(paddle::contrib::MixedRTConfig const&)'
[01:56:50]	[Step 5/5] collect2: error: ld returned 1 exit status
[01:56:50]	[Step 5/5] make[2]: *** [vis_demo] Error 1
[01:56:50]	[Step 5/5] make[1]: *** [CMakeFiles/vis_demo.dir/all] Error 2
[01:56:50]	[Step 5/5] make: *** [all] Error 2
[01:56:50]	[Step 5/5] ++ for use_gpu in '$use_gpu_list'
[01:56:50]	[Step 5/5] ++ for vis_demo_name in '$vis_demo_list'
[01:56:50]	[Step 5/5] ++ ./vis_demo --modeldir=/root/.cache/inference_demo/se_resnext50/model --data=/root/.cache/inference_demo/se_resnext50/data.txt --refer=/root/.cache/inference_demo/se_resnext50/result.txt --use_gpu=false
[01:56:50]	[Step 5/5] ./run.sh: line 87: ./vis_demo: No such file or directory
```
image

### PR DESCRIPTION
**Pull request by NHZlX**
fix demo_ci bug in vis_demo.cc
fix #13841 

The vis_demo.cc have the code of tensorrt, this will tigger bug when compile the paddle_fluid without tensorrt.  

http://ci.paddlepaddle.org/viewType.html?buildTypeId=Manylinux1_Cuda80cudnn5avxOpenblas&branch_Manylinux1=13848&tab=buildTypeStatusDiv

### CODE REVIEW

File: paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+/*
+ * This file contains demo of mobilenet for tensorrt.
+ */
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>  // use glog instead of CHECK to avoid importing other paddle header files.
+#include <fstream>
+#include <iostream>
+
+// #include "paddle/fluid/platform/enforce.h"
```
Comment (luotao1) [inline paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc]: 24行可以去掉

File: paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#include "paddle/fluid/inference/demo_ci/utils.h"
+
+#ifdef PADDLE_WITH_CUDA
```
Comment (luotao1) [inline paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc]: 27行可以去掉，保留28行即可，因为这个文件必须在gpu下运行

File: paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    data, "",
+    "path of data; each line is a record, format is "
+    "'<space splitted floats as data>\t<space splitted ints as shape'");
+
+namespace paddle {
+namespace demo {
+
+/*
+ * Use the native fluid engine to inference the demo.
+ */
```
Comment (luotao1) [inline paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc:36]: 41行注释不对，不是native engine

File: paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+limitations under the License. */
+
+/*
+ * This file contains demo of mobilenet for tensorrt.
+ */
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>  // use glog instead of CHECK to avoid importing other paddle header files.
+#include <fstream>
+#include <iostream>
```
Comment (luotao1) [inline paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc]: 21 22行头文件在util.h里有了，可以去掉

Comment (NHZlX) [reply paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc]: Done

File: paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc (hunk 3 above)
Comment (NHZlX) [reply paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc:36]: Done

File: paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc (hunk 2 above)
Comment (NHZlX) [reply paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc]: Done

File: paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc (hunk 1 above)
Comment (NHZlX) [reply paddle/fluid/inference/api/demo_ci/trt_mobilenet_demo.cc]: Done

**Comment by NHZlX** [comment]
http://ci.paddlepaddle.org/viewType.html?buildTypeId=Manylinux1_Cuda80cudnn5avxOpenblas&branch_Manylinux1=13848&tab=buildTypeStatusDiv

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**